newrelic-agent/src/main/java/com/newrelic/bootstrap/BootstrapAgent.java

```diff
@@ -88,7 +93,19 @@ static String decodeAndDecompressAgentArguments(String agentArgs) throws IOExcep
      */
     public static void premain(String agentArgs, Instrumentation inst) {
         String javaSpecVersion = JavaVersionUtils.getJavaSpecificationVersion();
-        if (!JavaVersionUtils.isAgentSupportedJavaSpecVersion(javaSpecVersion)) {
+        String sysExperimentalRuntime = System.getProperty("newrelic.config.experimental_runtime");
+        String envExperimentalRuntime = System.getenv("NEW_RELIC_AGENT_EXPERIMENTAL_RUNTIME");
```

Comment:
Same comment as the previous regarding `System.getProperty` and `System.getenv`.

Reply:
Both  System.getProperty() and System.getenv() are effectively guarded by Boolean.parseBoolean()..because if parseBoolean(null) takes a null parameter then it will return false.  

` 
   public static boolean parseBoolean(String s) {
        return ((s != null) && s.equalsIgnoreCase("true"));
    }
    
`